packages/networkx/meta.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  name: networkx
+  version: '2.2'
```

Comment:
Quotes should not be necessary

Reply:
I ran into issues without them - because there's no 'patch' version on networkX, yaml interprected 2.2 as a float, which prevented the package from building